### ISSUE TITLE
Frozen pfd and mfd after activating approach

### ISSUE THREAD

**Issue by tchiari** [Working-Title-MSFS-Mods/fspackages#708]
Both screens frozen so I couldn't read instruments nor make any changes. Was arriving SBRP and activated vor/dme to rwy 36 through RPR transition. Landed using steam gauges only. I was using the WT g1000 0.3.4 and the bonanza G36 0.61. I could still see the little FPL and APP popups on the screens when pressed those buttons but couldn't do any changes or select other options.

**Comment by kulibali** [comment]
For me in the DA-60 the map screen freezes when I activate an approach, but the PFD still works, and the flight plan can still be followed (it doesn't show up in the PFD inset map, but it appears in the VFR map).

**Comment by orenc** [comment]
same here! please fix!!!! lost a great event on vatsim in midflight... :(

**Comment by tchiari** [comment]
> same here! please fix!!!! lost a great event on vatsim in midflight... :(

Yeah but as far as I can see this isn’t exclusive to the working title, it’s the main simulator fault so instead of a fix we should expect for a workaround until ms or asobo toward their concerns to ifr instead of eye candy features

**Comment by kulibali** [comment]
Not so, the default G1000 does not freeze when activating an approach, and the Working Title mod worked fine prior to 0.3.4.

**Comment by clarktx** [comment]
Same here!  Please fix!  Great mod!  Thanks!

**Comment by Crunchmeister1** [comment]
Same for me.

**Comment by kmoze2001** [comment]
I submitted nearly an identical issue here:  https://github.com/Working-Title-MSFS-Mods/fspackages/issues/748
